src/worker-helpers.js

```diff
@@ -131,15 +131,15 @@ export function getConfigForFile(eslint, filePath) {
   try {
     return cli.getConfigForFile(filePath)
   } catch (e) {
-    // No configuration was found
-    return null
+    console.error('No configuration was found by eslint.CLIEngine, falling back to linter-eslint method.', e)
+    // eslint-disable-next-line no-use-before-define
```

Comment:
I feel like we should just move the function above this or disable the rule of we aren't going to follow it.

Reply:
Yes, I was planning to use eslint-config-atomic. These strict rules set by Airbnb limit me without adding any benefits to the project.

But it seems that our tests rely on the eslint config in `package.json`. So, we need to fix that before changing the eslint config.

For now, I think disabling these rules doesn't hurt. Later we can get rid of them.